librad/src/git/storage.rs

```diff
@@ -358,8 +358,17 @@ impl Storage {
             "Storage::track"
         );
 
-        let _ = self.backend.remote(&remote_name, &url)?;
-        Ok(())
+        match self.backend.remote(&remote_name, &url) {
```

Comment:
```suggestion
        self.backend
            .remote(&remote_name, &url)
            .map(|_| ())
            .map_not_found(|| Ok(()))
```

Reply:
Noice

Reply:
Actually it'll be `already_exists`, so I'm adding that to the extensions